presidio-analyzer/Dockerfile

```
@@ -26,7 +26,7 @@ RUN apt-get update \
 
 COPY ./pyproject.toml /app/
 
-RUN pip install poetry \
+RUN pip install poetry==2.3.2 \
```

Hadolint (2.14.0)

[warning] Quote this to prevent word splitting.

(SC2046)